docs/pages/guides/configuring-js-engines.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Configuring JS Engines
+sidebar_title: Configuring JS Engines
+description: A guide on configuring JS Engines for both Android and iOS in an Expo project.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+JavaScript engines are responsible for executing your application code and providing various features like memory management, optimization, and error handling. Expo projects, by default, use [Hermes](https://hermesengine.dev/) as the JavaScript engine (in SDK 47 and lower, the default was JSC). It is also possible to switch to other engines like JSC or V8 for both Android and iOS platforms, but not for web because the JavaScript engine is included in the web browser.
+
+### Configuring the `jsEngine` through Expo config
+
+We recommend Hermes because it is purpose built and optimized for React Native apps, and it has the best debugging experience. If you are familiar with the tradeoffs of different JavaScript engines and would like to change away from Hermes, the `jsEngine` field inside **app.json** allows you to specify the JavaScript engine for your app.
+
+The default value is `hermes`. If you would like to use [JavaScriptCore](https://developer.apple.com/documentation/javascriptcore) instead, set the `jsEngine` field in your **app.json**:
+
+```json
+{
+  "expo": {
+    "jsEngine": "jsc"
+  }
+}
+```
+
+<Collapsible summary="Are you using the bare workflow?">
+
+To change the JavaScript engine in a bare React Native project, update the `expo.jsEngine` value in **ios/Podfile.properties.json** and **android/gradle.properties**
+
+</Collapsible>
+
+It's important to highlight that changing the JS engine will require you to recompile development builds with `eas build` to work properly.
+
+### Using the V8 Engine
+
+To use the V8 engine, you need to install [react-native-v8](https://github.com/Kudo/react-native-v8), an opt-in package that adds V8 runtime support for React Native. You can install it by running the following command:
+
+<Terminal cmd={['$ npx expo install react-native-v8 v8-android-jit']} />
+
+Make sure to remove the `jsEngine` field from **app.json**
+
+<Collapsible summary="Are you using the bare workflow?">
+
+You can run `expo prebuild -p android --clean` after the installation to prebuild again.
+
+</Collapsible>
+
+### Switch JavaScript engine on a specific platform
+
+You may want to use a different engine on one specific platform, one way to do this is to set one `"jsEngine"` value at the top level and then override it with a different value under the `"android"`/`"ios"` key.
```

Comment:
```suggestion
To use a different engine on one specific platform, you can set `the "jsEngine"` value at the top level and then override it with a different value under `the "android"` or `"ios"` key. The value specified for the platform will take precedence over the common field.
```

Let's be descriptive about what the overriding value can do here. I see that you have highlighted it in the example already 👍 So thought we should add it here too.

We can also be more opinionated about the way we want developers to do things. For example, I removed the "one way to do..." context. This provides a vote of confidence from us (the Expo team) on how to do things in a general scenario.

Reply:
Thanks for rephrasing this Aman, let me just confirm one thing with you, do we want to put